packages/gatsby-theme-aio/src/components/Tabs/index.js

```diff
@@ -247,22 +255,38 @@ const TabsBlock = ({ theme = 'light', orientation = 'horizontal', className, ...
                 const ref = createRef();
                 tabs.push(ref);
                 const isSelected = selectedIndex.tab === index;
-                const itemPopoverId = nextId();
                 return (
                   <Item
                     className={'tabItem'}
                     key={`tabItem_${index}`}
-                    tabIndex={0}
+                    id={`tabItem_${index}`}
                     ref={ref}
                     isSelected={isSelected}
-                    aria-controls={itemPopoverId}
+                    aria-controls={`tabView${index}`}
+                    tabIndex={index === selectedIndex.tab ? 0 : -1}
+                    aria-label={data['heading']}
+                    aria-selected={index === selectedIndex.tab}
                     label={<b>{data['heading']}</b>}
                     icon={data['image']}
+                    onKeyDown={(e) => {
+                      if (e.key === 'ArrowDown' || e.key === 'Enter') {
+                        e.preventDefault();
+                        if (menuItems.length === index + 1 && APIReference !== '') {
+                          document.getElementById('apiReference')?.setAttribute('tabIndex', 0);
```

Comment:
Just wondering, where does the id `apiReference` come from ?

Reply:
Thanks for pointing this out! The `apiReference` section doesn't belong here since it's a custom functionality the DC team implemented. I was testing their keyboard support implementation and forgot I added it to the `Tabs` component as well. Will push a fix in a minute.